src/ava-fast-check.ts

```diff
@@ -1,17 +1,55 @@
-import test, { ExecutionContext, Implementation, ImplementationResult, TryResult } from 'ava';
+import test, {
+  AfterInterface,
+  BeforeInterface,
+  ExecutionContext,
+  Implementation,
+  ImplementationResult,
+  TestInterface,
+  TryResult
+} from 'ava';
 import * as fc from 'fast-check';
 
+export { fc, test };
+
+type NonEmptyArray<A> = A[] & {0: A};
+
 // Pre-requisite: https://github.com/Microsoft/TypeScript/pull/26063
 // Require TypeScript 3.1

```

Comment:
Not sure we still need this comment

Reply:
Agreed, it's descriptive of mapped types but not all that useful here. I will take it out